packages/client/src/components/pages/home-page/home-page.tsx

```diff
@@ -139,14 +139,16 @@ export const HomePage = () => {
         await dispatch(Workspace.fetchDashboards(orgId, workspace.id));
       }
 
-      try {
-        await dispatch(OrphanedRecordActions.fetchCount({ orgId: orgId! }));
-      } catch (error) {
-        void dispatch(Modal.alert('Get Orphaned Records', formatErrorMessage(error, 'Failed to get orphaned records')));
+      if (user.activeRole?.role.canManageRoster) {
```

Comment:
You can potentially make some of these permissions checks a little cleaner by getting the permissions the same way the View component does, like this...

```typescript
  const { canManageGroup } = useAppSelector(UserSelector.role)!;
```

Reply:
I am going to leave this one for now, since all the other checks in the file do not follow that syntax yet.